scripts/install.sh

```diff
@@ -529,6 +529,23 @@ elif [ "$format" = "tar" ]; then
       fi
     fi
 
+    # support creating /usr/local/bin if it's in the PATH but doesn't exist.
+    # this fixes an issue with clean installs of macOS 12+
+    install_dir="/usr/local/bin"
+    if [ ! -e "$install_dir" ] && [ "$( is_dir_in_path "$install_dir" )" -eq "1" ]; then
+      log_debug "/usr/local/bin is in PATH but doesn't exist"
+      log_debug "Creating /usr/local/bin"
+      if mkdir -m 755 "$install_dir" > /dev/null 2>&1 ; then
```

Comment:
Would it be worthwhile to add the `-p` flag as well, in the event that the parent dirs also don't exist?

Reply:
I don't think so- if `/usr/local/` doesn't exist there's a bigger problem.